packages/core/src/index.ts

```diff
@@ -53,7 +54,10 @@ export {
 export type * from "./client/decorators/bundlerClient.js";
 export { bundlerActions } from "./client/decorators/bundlerClient.js";
 export type * from "./client/decorators/smartAccountClient.js";
-export { smartAccountClientActions as smartAccountClientDecorator } from "./client/decorators/smartAccountClient.js";
+export {
+  smartAccountClientActions,
+  smartAccountClientActions as smartAccountClientDecorator,
```

Comment:
any reason we need to export it twice under two namespaces?

Reply:
oh yea I missed this too good question